employee-rostering-backend/src/main/java/org/optaweb/employeerostering/domain/shift/Shift.java

```diff
@@ -33,7 +34,10 @@
 
 @Entity
 @PlanningEntity
-public class Shift extends AbstractPersistable {
+public class Shift extends AbstractPersistable implements Comparable<Shift> {
+
+    private static final Comparator<Shift> PILLAR_SEQUENCE_COMPARATOR = Comparator.comparing(Shift::getStartDateTime)
```

Comment:
`PILLAR_SEQUENCE_COMPARATOR` is not a good name (and I argue code for the benchmarker should be in the benchmarker). If you need to have a `compareTo`, a name like `TIME_SLOT_COMPARATOR` would be better.

Reply:
The subpillar solver configuration requires the `Shift` class to implement the `compareTo()` method (like in the [NurseRostering OptaPlanner example](https://github.com/kiegroup/optaplanner/blob/master/optaplanner-examples/src/main/java/org/optaplanner/examples/nurserostering/domain/ShiftAssignment.java#L118)). I don't see a way to separate this from the `Shift` class, which is in the backend. This method isn't necessarily Benchmark code, it just specifies a way to compare `Shift` objects, so I think it's fine in the backend module. WDYT?

Reply:
From the docs (https://docs.optaplanner.org/7.27.0.Final/optaplanner-docs/html_single/index.html#pillarMoveSelectors), there a way to specify a comparator WITHOUT needing to implement `compareTo`: ` <subPillarSequenceComparatorClass>org.optaplanner.examples.nurserostering.domain.ShiftAssignmentComparator</subPillarSequenceComparatorClass>`